service_restaurant/src/main/kotlin/org/thechance/service_restaurant/api/models/MealDto.kt

```diff
@@ -0,0 +1,12 @@
+package org.thechance.service_restaurant.api.models
+
+import kotlinx.serialization.Serializable
+
+@Serializable
+data class MealDto(
+    val id: String? = null,
+    val name: String? = null,
+    val description: String? = null,
+    val price: Int? = null,
+    val isDeleted: Boolean = false
```

Comment:
i think there is no need to show for client  if meal isDeleted from database or not